packages/core/utils/src/replaceBundleReferences.js

```diff
@@ -33,13 +33,15 @@ export function replaceURLReferences({
   bundleGraph,
   contents,
   map,
+  getReplacement,
   relative = true,
 }: {|
   bundle: NamedBundle,
   bundleGraph: BundleGraph<NamedBundle>,
   contents: string,
   relative?: boolean,
   map?: ?SourceMap,
+  getReplacement: string => string,
```

Comment:
This is technically a breaking change, though the question is how many plugins are out there using this function.